src/Microsoft.DotNet.XHarness.CLI/CommandArguments/WASM/Arguments/LocaleArgument.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.XHarness.CLI.CommandArguments.Wasm;
+using System;
+
+internal class LocaleArgument : StringArgument
+{
+    public LocaleArgument(string defaultValue)
```

Comment:
I think you're not using the `defaultValue` in any way? Have a look at `RequiredStringArgument` and pass it to its ctor.